packages/crayons-extended/custom-objects/src/components/form-builder/components/field-editor.scss

```diff
@@ -262,6 +262,28 @@
           height: auto;
           box-sizing: border-box;
         }
+
+        .fw-field-editor-status-toggle {
+          border: 1px solid #cfd7df;
+
+          .fw-field-editor-status-toggle-item {
+            display: grid;
+            font-size: 14px;
```

Comment:
Please use scss variables instead of hard-coding the values.

Reply:
Updated